### PR TITLE
release: 0.7.0

### PR DESCRIPTION
**Pull request by tiborsimko**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,25 +1,47 @@
...
-- Stops exposing unused Invenio-Accounts views.
-- Starts up Postgres DB container to run tests for componentes installing reana-db.
+Version 0.7.0 (2020-10-19)
+--------------------------
+
+- Users:
+    - Adds option to specify unpacked Docker images as workflow step requirement.
+    - Adds option to specify Kubernetes UID for jobs.
+    - Adds support for VOMS proxy as a new authentication method.
+    - Adds support for pulling private images using image pull secrets.
```
Comment (diegodelemos) [inline CHANGES.rst:11]: Maybe we can just mention the feature without the implementation detail?
```suggestion
    - Adds support for pulling private images.
```

File: CHANGES.rst [hunk 2]
```diff
@@ -1,25 +1,47 @@
...
+    - Adds user profile page to the Web interface.
```
Comment (diegodelemos) [inline CHANGES.rst:12]:
```suggestion
    - Adds user profile page to the web interface.
```

File: CHANGES.rst [hunk 3]
```diff
@@ -1,25 +1,47 @@
...
+    - Fixes workflow specification display to show runtime parameters.
+    - Changes CERN HTCondor compute backend to use the new ``myschedd`` connection library.
+    - Changes CERN Slurm compute backend to improve job status detection.
+    - Changes documentation to move large parts to docs.reana.io.
+    - Changes ``du`` command output format.
+    - Changes file loading to optimise CLI performance.
+    - Changes ``logs`` command to enhance formatting using marks and colours.
+    - Changes from Bravado to requests to improve download performance.
+    - Changes ``ping`` command to perform user access token validation.
+    - Changes defaults to accepts both ``reana.yaml`` and ``reana.yml`` filenames.
```
Comment (mvidalgarcia) [inline CHANGES.rst:27]:
```suggestion
    - Changes defaults to accept both ``reana.yaml`` and ``reana.yml`` filenames.
```

File: CHANGES.rst [hunk 4]
```diff
@@ -1,25 +1,47 @@
...
+    - Adds user profile page to the Web interface.
+    - Adds pagination on the workflow list web page.
+    - Adds option to ``logs`` command to filter job logs according to compute backend, docker image, status and step name.
+    - Adds new ``restart`` command to restart previously run or failed workflows.
+    - Adds option to specify operational options in the ``reana.yaml`` of the workflow.
+    - Fixes file preview functionality experience to allow/disallow certain file formats.
+    - Fixes workflow specification display to show runtime parameters.
+    - Changes CERN HTCondor compute backend to use the new ``myschedd`` connection library.
+    - Changes CERN Slurm compute backend to improve job status detection.
+    - Changes documentation to move large parts to docs.reana.io.
```
Comment (diegodelemos) [inline CHANGES.rst:21]: Should we add an actual link to https://docs.reana.io?

**Review by mvidalgarcia — APPROVED** [review]

File: setup.py [hunk 5]
```diff
@@ -19,7 +19,7 @@
 history = open("CHANGES.rst").read()
 
 tests_require = [
-    "pytest-reana>=0.7.0.dev20200707",
+    "pytest-reana>=0.7.0a1",
```
Comment (diegodelemos) [inline setup.py:22]:
```suggestion
    "pytest-reana>=0.7.0",
```

File: CHANGES.rst [hunk 6]
```diff
@@ -1,25 +1,47 @@
...
+    - Changes file loading to optimise CLI performance.
+    - Changes ``logs`` command to enhance formatting using marks and colours.
+    - Changes from Bravado to requests to improve download performance.
+    - Changes ``ping`` command to perform user access token validation.
+    - Changes defaults to accepts both ``reana.yaml`` and ``reana.yml`` filenames.
+    - Changes ``diff`` command to improve output formatting.
+- Administrators:
+    - Adds Helm chart and switches to Helm-based deployment technique instead of using now-deprecated ``reana-cluster``.
+    - Adds email notification service to inform administrators about system health.
+    - Adds announcement configuration option configuration to display any desired text on the Web UI.
```
Comment (diegodelemos) [inline CHANGES.rst:32]: It is not clear to me when we use the capital letters or not (see https://github.com/reanahub/reana/pull/426/files#r508640338 and https://github.com/reanahub/reana/pull/426/files#diff-ff3c479edefad986d2fe6fe7ead575a46b086e3bbcf0ccc86d85efc4a4c63c79R13) so maybe the initial comment didn't make sense...

**Review by diegodelemos — APPROVED** [review]

File: CHANGES.rst [hunk 7]
```diff
@@ -1,25 +1,47 @@
...
+    - Adds support for local user management and web forms for sign-in and sign-up functionalities.
+    - Changes installation procedures to move database initialisation and admin creation after Helm installation.
+    - Changes service exposure to stop exposing unused Invenio-Accounts views.
+    - Changes runtime job instantiation into the configured runtime namespace.
+    - Changes CVMFS to be read-only mount.
+- Developers:
+    - Adds several new ``reana-dev`` commands to help with merging, releasing, unit testing.
+    - Changes base image to use Python 3.8 for all REANA cluster components.
+    - Changes code formatting to respect ``black`` coding style.
+    - Changes test strategy to start PostgreSQL DB container to run tests locally.
```
[the remark this answers is not in the document]
Comment (tiborsimko) [reply CHANGES.rst:43]: Added line to the admin part:

-  Adds support for database upgrades using Alembic.

File: CHANGES.rst (hunk 6 above)
Comment (tiborsimko) [reply CHANGES.rst:32]: Yeah, both are kind of possible... downcased to `web UI`.

**Pull request merged**